components/table/src/Table.js

```diff
@@ -8,157 +8,158 @@ import { jsx, useTheme } from '@westpac/core';
 // Utils
 // ==============================
 
-const TableWrapper = ({ isBordered, isResponsive, children, ...props }) => {
-	const { table, breakpoints } = useTheme();
+const TableWrapper = ({ bordered, responsive, ...props }) => {
+	const {
+		COLORS,
+		LAYOUT: { breakpoints },
+	} = useTheme();
 
 	const maxWidth = width => `@media (max-width: ${width}px)`;
 	const xsOnly = maxWidth(breakpoints.sm - 1);
 
-	const common = {
-		[xsOnly]: {
-			width: '100%',
-			marginBottom: '1.8rem',
-			overflowY: 'hidden',
-			overflowX: 'auto',
-			// -ms-overflow-style: '-ms-autohiding-scrollbar',
-			border: `${table.responsive.borderWidth} solid ${table.responsive.borderColor}`,
-			// -webkit-overflow-scrolling: 'touch',
-
-			'> table': {
-				marginBottom: 0,
-				border: isBordered && 0,
-
-				caption: {
-					padding: table.responsive.caption.padding,
-				},
-				'tbody, tfoot': {
-					'tr:last-child': {
+	return responsive ? (
+		<div
+			className="table-responsive"
```

Comment:
This will change (we don't want to rely on a className to change styles managed in another component), but it will be easier to merge both the `Table` and `Panel` component changes in `gel-develop`, and open a new PR to _decouple_ them.